tests/favicon.spec.ts

```diff
@@ -57,3 +73,48 @@ it('should load svg favicon with prefer-color-scheme', async ({page, server, bro
   // Text still being around ensures we haven't actually lost our browser to a crash.
   await page.waitForSelector('text=favicons');
 });
+
+it('should filter favicon and favicon redirects from request logs', async ({server, browserName, channel, headless, asset, contextFactory}, testInfo) => {
```

Comment:
since this is about the HAR feature, I would maybe move it into the har.spec.ts

Reply:
👍 good call